setup-tdx-guest.sh

```diff
@@ -24,7 +24,7 @@ apt install --yes software-properties-common gawk &> /dev/null
 rm -f /etc/apt/preferences.d/kobuk-team-tdx-*
 rm -f /etc/apt/apt.conf.d/99unattended-upgrades-kobuk
 
-add_kobuk_ppa
+add_kobuk_ppa ${TDX_PPA:-tdx}
```

Comment:
Should the default here be `tdx-release`, since that's mentioned as the default in the configuration file? I guess the scenario where TDX_PPA is not defined should not happen, but if we're using a fallback here it seems like `tdx-release` would be more consistent.

Reply:
yes, you are right, thanks